metabrainz/admin/views.py

```diff
@@ -281,25 +281,41 @@ def lookup_ips(users):
 
     @expose('/top-ips/')
     def top_ips(self):
-        non_commercial, commercial = AccessLog.top_ips(limit=100, days=7)
+
+        days = 7
```

Comment:
Maybe this could be shared in a single function?

Reply:
I'll do that for the next round to changes -- there will be quite a few more reports coming.